pages/common/nasm.md

```diff
@@ -0,0 +1,23 @@
+# nasm
+
+> The Netwide Assembler, a portable 80x86 assembler.
+
+- Assemble raw binary file `source` from `source.asm`:
+
+`nasm {{source.asm}}`
+
+- Assemble and generate an assembly listing file:
+
+`nasm -l {{list_file}} {{source.asm}}`
+
+- Add a directory to the include file search path before assembling:
+
+`nasm -i {{/path/to/include_dir}} {{source.asm}}`
+
+- Assemble binary file `output_file` of the specified format:
+
+`nasm -f {{format}} {{source.asm}} -o {{output_file}}`
```

Comment:
I would put this as the second example, since the "source" in the output filename of the first example threw me off for a second there. But I'm open to consider reasons why it makes more sense to place this example here.